examples/complex/test/Donations.behavior.js

```diff
@@ -0,0 +1,115 @@
+import assertRevert from 'zos-lib/src/helpers/assertRevert';
+
+const getBalance = require('./helpers/getBalance.js');
+const should = require('chai').should();
+
+module.exports = function(owner, donor, wallet) {
+
+  describe('donate', function () {
+
+    describe('when the donation is zero', function() {
+
+      let donation;
```

Comment:
You can remove the `let` declaration here, and use `this.donation`. It's a matter of personal preference though.